README.rst

```diff
@@ -89,7 +89,7 @@ Example installation command using Pathogen:
 
 .. code-block:: sh
 
-    cd ~/.vim/bundle/ && git clone --recursive https://github.com/davidhalter/jedi-vim.git
+    cd ~/.vim/bundle/ && git clone --recursive https://github.com/davidhalter/jedi-vim.git && cd -
```

Comment:
What about `git clone --recursive https://github.com/davidhalter/jedi-vim.git ~/.vim/bundle/jedi-vim` instead?

Reply:
Wow, it totally slipped my mind. This is a perfect one.